README.md

```diff
@@ -1 +1,124 @@
-# my-catalog
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [My Catalog App] <a name="about-project"></a>
+
+### Video description
+
+[CLick to watch video](https://drive.google.com/file/d/1YLngZXukysB0ou5tGQvS_QXsk7qR7OG9/view?usp=sharing)
+
```

Comment:
- [ ] Please kindly ensure that this presentation is not more than 5minutes as stated in the requirement. 🙏 

<img width="780" alt="Untitled" src="https://github.com/martinkarugaba/my-catalog/assets/103386183/a7c17fff-6248-4179-983c-3a61655d790b">